doc/cephadm/operations.rst

```diff
@@ -734,3 +734,64 @@ Purge ceph daemons from all hosts in the cluster
 
   # For each host:
   cephadm rm-cluster --force --zap-osds --fsid <fsid>
+
+
+Replacing a device
+==================
+
+The ``ceph orch device replace`` command automates the process of replacing the underlying device of an OSD.
+Previously, this process required manual intervention at various stages.
+With this new command, all necessary operations are performed automatically, streamlining the replacement process
+and improving the overall user experience.
+
+.. note:: This only supports LVM-based deployed OSD(s)
+
+.. note:: This will work only if ``ceph-volume lvm list`` sees the corresponding OSD(s)
```

Comment:
As discussed over slack, this actually did confuse me, we should either remove that, or explain `what` would make ceph-volume lvm list `not see` an OSD. (that would be actually probably only `raw` deployments, so maybe even redundant given that above we state that it is only supported by lvm-based deployments, which I understand to be either manually created LVM setups or ceph-volume automatically created LVM setups.

Reply:
I think I would remove that comment. I'm not sure we should go too deep into details here.

Reply:
I think this comment might have only been added because I was asking for clarification on what conditions need to be met for this to work earlier. At the time I thought those conditions were more strict than they really are, so I'd be okay with removing this as well.